### DISCUSSION TITLE
Navier slip boundary condition

### DISCUSSION THREAD

**Discussion by irwinosmond1983** [fppimenta/rheoTool#80] [General]
Navier slip boundary condition is not working with rheoInterFoam however its works fine with other solver. Please help. 
Getting this error
 FOAM FATAL ERROR: 

    lookup of constitutiveProperties from objectRegistry region0 successful
    but it is not a constitutiveModel, it is a dictionary

    From function const Type& Foam::objectRegistry::lookupObject(const Foam::word&) const [with Type = Foam::constitutiveModel]
    in file /opt/openfoam7/src/OpenFOAM/lnInclude/objectRegistryTemplates.C at line 178.

**Comment by fppimenta** [comment]
isTwoPhaseFlow = true

**Reply by irwinosmond1983** [reply]
I want to implement navier slip as:    slip length x du/dx (velocity gradient in tangential direction) , I am slightly confused how to choose the coefficients in this boundary condition . Please help

**Comment by fppimenta** [comment]
The BC is documented in the user-guide.

**Reply by irwinosmond1983** [reply]
Thanks , I got that . rheoTool is not having fe05 version ??

**Comment by fppimenta** [comment]
No, the foam-extend branch is not even up to date.
